monitor/src/flow/index.ts

```diff
@@ -0,0 +1,24 @@
+import { handleAPIKey } from './handleAPIKey';
+import { handleAction } from './handleAction';
+import { handleSessionToken } from './handleSessionToken';
+import { handleTask } from './handleTask';
+import { handleResult } from './handleResult';
+import { handleValidation } from './handleValidation';
+import { handleRecord } from './handleRecord';
+import { handleCredentials } from './handleCredentials';
+import { handleError } from './handleError';
+import { RequestConfig, MonitorResponse } from '../types';
+
+export const handleFlow = async (config: RequestConfig): Promise<MonitorResponse> => {
+  const start = process.hrtime();
+  const result = await handleAPIKey(config)
+    .then((response) => handleAction(response))
```

Comment:
```suggestion
    .then(handleAction)
```
Ídem para las funciones de abajo que sólo reciben response como parámetro

Reply:
done